js/aside-script.js

```diff
@@ -211,6 +211,22 @@ function UpdateLocale()
 	);
 }
 
+chrome.storage.onChanged.addListener((changes, namespace) =>
```

Comment:
Having this listener in the aside script, as opposed to the background script, means no updates will be done if the pane is not open. So, if the user sets aside tabs without opening the pane (shortcuts, context menu, or something else), then the collections variable remains out of sync with the chrome storage, and the chrome storage is overwritten with the collections variable.